ragna/core/_rag.py

```diff
@@ -82,21 +83,34 @@ def _load_component(
 
     def chat(
         self,
-        input: Union[MetadataFilter, None, Iterable[Union[Document, str, Path]]],
+        input: Union[
+            MetadataFilter,
+            None,
+            Document,
+            str,
+            Path,
+            Collection[Union[Document, str, Path]],
+        ],
         *,
         source_storage: Union[Type[SourceStorage], SourceStorage],
         assistant: Union[Type[Assistant], Assistant],
-        corpus_name: Optional[str],
+        corpus_name: Optional[str] = None,
```

Comment:
@nenb I've added a default value here, because I don't want to force users to always set this one.